assignments/arrays.js

```diff
@@ -63,34 +64,49 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log('Car 33 is a ' + inventory[33 - 1].car_year + ' '+ inventory[33 - 1].car_make + ' ' + inventory[33 - 1].car_model );
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+console.log(inventory[inventory.length - 1].car_make + ' ' + inventory[inventory.length - 1].car_model);
```

Comment:
This works! You could also save yourself some typing by assigning lastCar to be the element at the last index. Then you could do `lastCar.car_make` for example. But the way you did it is effectively the same!